### DISCUSSION TITLE
Can I turn off Korean tokenizer?

### DISCUSSION THREAD

**Discussion by datamaker** [meilisearch/product#683] [Feedback & Feature Proposal]
We are stuck at version v0.29.
The search quality is not good because of the phenomenon that Hangul morpheme analysis is too finely divided.
However, I want to use all the features of v1.3, but I can't.

We believe that the search quality is not as good as it is currently based on whitespace.
https://github.com/meilisearch/product/discussions/623
https://github.com/meilisearch/product/discussions/564

Will it be accepted if I PR the ability to turn the tokenizer on and off with an environment variable?
https://github.com/meilisearch/meilisearch/discussions/3905

**Comment by ManyTheFish** [comment]
Hello @datamaker, it's unfortunate that the tokenization of Hangul doesn't suit your need, we want to have good support in the Korean Language, could you please do several tests on your side:
1) change the order of your ranking rules to `["word", "proximity", "exactness",  "typo", "attribute"]`
2) recompile Meilisearch with the following command line:
  a) `cargo build --release --no-default-features --features "analytics mini-dashboard"`
  b) `/target/release/meilisearch`
  c) reindex your dataset

 Let me know if it enhances your use case, and which one is the most efficient. Don't hesitate to give examples.
 
 Sorry for this inconvenience.
